config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -265,7 +273,7 @@ EOF
                         cat >>"$base/$repo/$repo-config.yaml" <<EOF
   - name: $(job_name "pull" "$repo" "$tests" "$deployment" "$kubernetes")
     always_run: $(pull_alwaysrun "$tests")
-    optional: $(pull_optional "$tests")
+    optional: $(if [ "$kubernetes" = "1.16.2" ]; then echo true; else pull_optional "$tests"; fi)
```

Comment:
This special case is not obvious and deserves a comment. Perhaps change this into `optional: $(pull_optional "$kubernetes" "$tests")` and then inside the extended implementation of `pull_optional` explain why 1.16.2 is an exception?

Reply:
done